### PR TITLE
docs: change zkquiz questions guide

### PR DESCRIPTION
**Pull request by avilagaston9**
**Motivation**

This PR adds a new guide to the documentation that explains how to modify the questions in the ZkQuiz trivia application to custom ones.

### CODE REVIEW

File: docs/3_guides/2.2_modify_zkquiz_questions.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# Modify ZkQuiz Questions
+
+In [Build your first Aligned Application](2_build_your_first_aligned_application.md), we show how to build a trivia application, called ZkQuiz. ZKQuiz asks the user three questions, and if answered correctly, generates a ZK Proof of the correct answers, posts the proof on Aligned, and upon verification, mints an NFT via a smart contract.
+
+In this guide, we will show you how to replace those questions with others.
```
Comment (IAvecilla) [inline docs/3_guides/2.2_modify_zkquiz_questions.md]: I would suggest something like this

```suggestion
In this guide, we will show you how to replace those questions with your own custom ones.
```

File: docs/3_guides/2.2_modify_zkquiz_questions.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+user_answers.push(ask_question(question2, &answers2));
+
+let question3 = "What is the native cryptocurrency of Ethereum?";
+let answers3 = ["Bitcoin", "Ether", "Litecoin"];
+user_answers.push(ask_question(question3, &answers3));
+```
+
+## 2. Update the Program
+
+Now we need to update the program to be proven with the new correct answers. As we saw in [Build your first Aligned Application](2_build_your_first_aligned_application.md), the program in `examples/zkquiz/quiz/program/src/main.rs` takes the user answers as inputs and checks that the hash of these inputs matches the expected output. Therefore, we need to update the expected output with the hash of our new correct answers. The updated example looks as follows:
```
Comment (IAvecilla) [inline docs/3_guides/2.2_modify_zkquiz_questions.md]: For this section, I suggest including a brief example of how you obtained the expected bytes for the output. It doesn't need to be working code to calculate the sha3 hash, but maybe just mention that this output is the result of hashing sha3("cab"), as those are the correct answers in this case.

File: docs/3_guides/2.2_modify_zkquiz_questions.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+            1, 186, 223, 215, 118, 206, 47, 12, 206, 114, 118, 220, 93,
+        ]
+    {
+        panic!("Answers do not match");
+    }
+```
+
+## 3. Compile the Program
+
+Now we need to compile the updated Program, generating the binary file that will be run by the zkVm (ELF).
```
Comment (IAvecilla) [inline docs/3_guides/2.2_modify_zkquiz_questions.md]:
```suggestion
Now we need to compile the updated Program, generating the binary file that will be run by the zkVM (ELF).
```

File: docs/3_guides/2.2_modify_zkquiz_questions.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+For that you can use the [SP1 Rust toolchain](https://docs.succinct.xyz/introduction.html).
+
+```
+cd examples/zkquiz/quiz/program
+cargo prove build
+```
```
Comment (IAvecilla) [inline docs/3_guides/2.2_modify_zkquiz_questions.md]: Add this as a Makefile target and explain that SP1 would be needed to run it.

File: docs/3_guides/2.2_modify_zkquiz_questions.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+In this guide, we will show you how to replace those questions with others.
+
+{% hint style="warning" %}
+This guide assumes you have already read [Build your first Aligned Application](2_build_your_first_aligned_application.md)
+{% endhint %}
+
+## 1. Modify the Questions Asked
+
+First, we need to modify the questions presented to the user. To do this, navigate to our example in `examples/zkquiz/quiz/script/src/main.rs` and change the questions as needed:
+
```
Comment (IAvecilla) [inline docs/3_guides/2.2_modify_zkquiz_questions.md:14]: I would add a brief disclaimer here to explain that this is a modified version and not the original. Something like: 'Here’s a new set of questions with their respective answers.'

File: docs/3_guides/2.2_modify_zkquiz_questions.md (hunk 2 above)
Comment (avilagaston9) [reply docs/3_guides/2.2_modify_zkquiz_questions.md]: Done!

File: docs/3_guides/2.2_modify_zkquiz_questions.md (hunk 4 above)
Comment (avilagaston9) [reply docs/3_guides/2.2_modify_zkquiz_questions.md]: Done!

File: docs/3_guides/2.2_modify_zkquiz_questions.md (hunk 5 above)
Comment (avilagaston9) [reply docs/3_guides/2.2_modify_zkquiz_questions.md:14]: Done!

**Review by PatStiles — APPROVED** [review]

**Review by IAvecilla — APPROVED** [review]

**Review by Oppen — APPROVED** [review]

**Review by MarcosNicolau — APPROVED** [review]

File: docs/SUMMARY.md [hunk 6]
```diff
@@ -25,6 +25,7 @@
 
 * [Submitting proofs](3_guides/0_submitting_proofs.md)
 * [Build your first Aligned Application](3_guides/2_build_your_first_aligned_application.md)
+* [Modify ZkQuiz Questions](3_guides/2.2_modify_zkquiz_questions.md)
```
Comment (JuArce) [inline docs/SUMMARY.md]: It could be added as a subitem of Build your first Aligned Application

**Review by JuArce — CHANGES_REQUESTED** [review]

File: docs/SUMMARY.md (hunk 6 above)
Comment (avilagaston9) [reply docs/SUMMARY.md]: Done!

**Review by JuArce — APPROVED** [review]

**Pull request merged**